src/sidebar/components/test/timestamp-test.js

```diff
@@ -31,7 +31,7 @@ describe('Timestamp', () => {
 
   it('displays a link if an "href" is provided', () => {
     const wrapper = createTimestamp({
-      timestamp: '',
+      timestamp: '2016-06-10',
```

Comment:
This fixes an issue where we ended up passing an invalid argument to the `Date` constructor which throws in Chrome.